### DISCUSSION TITLE
military_education_01_artesh/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#27] [Announcements]
# military_education_01_artesh/

در این پست می‌خوام تجربه دوره آموزشی که در آبان ۱۴۰۲ در پادگان ۰۱ ارتش گذروندم، رو براتون تعریف کنم. این پست به درد کسانی خواهد خورد که می‌خوان تو دوره آموزشی نخبگان شرکت کنند و می‌خوان بدونند چه چیزی در انتظارشون خواهد بود.
قبل از رفتن به دوره تجربه خیلی از بچه‌ها رو خونده بودم و توقعم بالا رفته بود. خصوصاً این که قدیمی‌ترها می‌گفتند: «۰۱ هتله». فایل صوتی‌ای که گوش دادم و خیالم از این بابت راحت‌تر شد، تجربه آموزشی 01 - اردیبهشت‌ماه 1402 بود.

https://aprd.ir/military_education_01_artesh/

**Comment by amirghh** [comment]
سلام، آخ آخ منم به امید هتل بودن 01 سال 87 رفتم اونجا. با وزن 79 رفتم و 59 کیلو برگشتم. با تعریفی که کردی خیلی فرق کرده. البته بگم من نخبه نبودم و معمولی حساب می شدم :-))
اما بدترین بخش سربازی من بود نه بخاطر تحرک و نظم خشکی که حاکم بود بلکه بخاطر گرمای شدید هوا (من تیر تا شهریور اونجا بودم)، نبود آب برای دوش گرفتن (در روز 2 ساعت آب بود)، عدم کفایت غذا برای سیر شدن (واقعا مدیون ساقه طلایی هستم) و...
عشق من این بود پست ها نگهبانی پرت رو بگیرم بعد سرنیزه رو قایم کنم و بگیرم تخت بخوابم تا صبح :-))
بعدشم که مجبور شدم دوره کد بگذرونم تو شیراز و بعد هم شدم افسر آموزش :-)

خلاصه 17 ماه عمرم به بطالت گذشت

**Reply by pourmand1376** [reply]
سلام. امیر جان. 

این‌طور که می‌گفتند آموزشی سرباز نخبه و سرباز امریه و سرباز عادی با هم فرق داره. حتی متوجه شدم که غذای ما هم با غذای سربازهای وظیفه فرق داشت. می‌گفتند غذای ما بهتره و کادری‌ها هم همون رو می‌خوردند. بعد من با خودم می‌گفتم که این غذاها، آشغال‌ترین و بی‌کیفیت‌ترین غذاهایی هست که تو عمرم خوردم. دیگه سرباز عادی‌ها چی می‌خوردند واقعا :( 

من این حجم تفاوت‌گذاشتن بین انواع مختلف سرباز رو هم نمی‌فهمم. تو مراسم‌های مختلف به ما می‌گفتند «ذخیره‌های نظام» یا نخبه یا اصطلاحاتی از این جنس؛ یعنی منظورشون اینه که اون سربازی که بالای برجک پست می‌ده ذخیره نظام نیست؟‌ اون به هیچ دردی نمی‌خوره؟ اون آدم نیست؟ 

تو این مورد دکتر رنانی یه مطلب مفصل نوشتند و صحبت کردند ([جوانان، آخرین سرمایه‌های ایران | تارنمای محسن رنانی](https://renani.net/texts/notes/iran-elite-pathology/)). تو دوره سربازی هی حرف‌هاشون برام مرور می‌شد و به کارهای بیهوده‌ای که بندگان خدا مجبور بودند، انجام بدند فکر می‌کردم. اصلاً اولش می‌خواستم موضوع پست رو بگذارم، «تجربه دوره آموزش بردگی نخبگانی در ارتش» که دیدم خیلی تند میشه.

**Comment by rezat7392-ops** [comment]
سلام اون لینک گروه تلگرام بالا نمیاد

**Reply by pourmand1376** [reply]
سلام. اون گروه پاک شد. الان یه گروه مشابه اینجا درست کردند. @IrEliteMedX

**Comment by mustafa-sadeghi** [comment]
سلام جناب پورمند
ممنون از به اشتراک گذاشتن تجربه تون.
آموزشی سربازان امریه هم شرایط مشابه سرباز نخبه داره؟ 
فقط آموزشی سرباز نخبه ها 1 ماهه هستش ؟

**Reply by pourmand1376** [reply]
سلام اقا مصطفی. 
تا اونجا که می‌دونم آموزشی امریه‌ها مثل عادی‌ها هست و تفاوتی قائل نمیشن. یادمه دوستانم دو ماهه رفته بودند و خیلی هم اذیت‌شون کرده بودند و خلاصه ناراضی بودند. 
البته از لحن پست می‌تونید متوجه بشید که من هم چندان از آموزشی خودمون راضی نبودم؛ ولی فکر کنم اون برمی‌گرده به توقع من از آموزشی نخبگان که گفته بودند هتل هست. وگرنه در کل - گرچه دوره‌ای سرتاسر بیهوده بود و جز دوست‌یابی کار دیگه‌ای نکردیم - ولی بدک نبود.
